config/database.yml

```diff
@@ -1,16 +1,18 @@
-development:
+default: &default
   adapter: postgresql
-  database: growstuff_dev
-  host: localhost
+  encoding: unicode
+  host: db
```

Comment:
Yes (it's currently breaking Travis). We'll need to think about that. It might be OK to tell existing contributors "the new preferred way to run a development environment is with Docker, please throw away your hand-configured environments".